package.json

```diff
@@ -13,9 +13,9 @@
   "dependencies": {
     "babel-polyfill": "^6.26.0",
     "expose-loader": "^0.7.5",
-    "genie": "git+ssh://git@github.com/bbc/childrens-games-genie.git#v0.4.1",
+    "genie": "git+ssh://git@github.com/bbc/childrens-games-genie.git#v0.5.0",
     "npm": "^5.7.1",
-    "phaser-ce": "2.10.5",
+    "phaser-ce": "^2.11.0",
```

Comment:
Do we actually need phaser as an explicit dependency? Shouldn't be installed as a dependency of genie? It would save having to keep the versions in sync.